src/Pages/TemplatesTable/components/AddTemplate/steps/RedhatRepositoriesStep.tsx

```diff
@@ -0,0 +1,324 @@
+import { createUseStyles } from 'react-jss';
+import { ContentItem, ContentOrigin } from '../../../../../services/Content/ContentApi';
+import { useState } from 'react';
+import { useContentListQuery } from '../../../../../services/Content/ContentQueries';
+import { SearchIcon } from '@patternfly/react-icons';
+import EmptyTableState from '../../../../../components/EmptyTableState/EmptyTableState';
+import { useHref } from 'react-router-dom';
+import Hide from '../../../../../components/Hide/Hide';
+import { SkeletonTable } from '@patternfly/react-component-groups';
+import { Table, TableVariant, Tbody, Td, Th, ThProps, Thead, Tr } from '@patternfly/react-table';
+import UrlWithExternalIcon from '../../../../../components/UrlWithLinkIcon/UrlWithLinkIcon';
+import PackageCount from '../../../../ContentListTable/components/PackageCount';
+import useDebounce from '../../../../../Hooks/useDebounce';
+import { REPOSITORIES_ROUTE } from '../../../../../Routes/constants';
+import TdWithTooltip from '../../../../../components/TdWithTooltip/TdWithTooltip';
+import { reduceStringToCharsWithEllipsis } from '../../../../../helpers';
+import ConditionalTooltip from '../../../../../components/ConditionalTooltip/ConditionalTooltip';
+
+const useStyles = createUseStyles({
+  global_300: {
+    color: global_Color_300.value,
+  },
+  topBottomContainers: {
+    justifyContent: 'space-between',
+    height: 'fit-content',
+  },
+  invisible: {
+    opacity: 0,
+  },
+});
+
+export default function RedhatRepositoriesStep() {
+  const classes = useStyles();
+  const path = useHref('content');
+  const pathname = path.split('content')[0] + 'content';
+
+  const {
+    hardcodedRedhatRepositoryUUIDS,
+    templateRequest,
+    selectedRedhatRepos,
+    setSelectedRedhatRepos,
+  } = useAddTemplateContext();
+
+  const [toggled, setToggled] = useState(false);
+
+  const setUUIDForList = (uuid: string) => {
+    if (selectedRedhatRepos.has(uuid)) {
+      selectedRedhatRepos.delete(uuid);
+      if (selectedRedhatRepos.size - hardcodedRedhatRepositoryUUIDS.size === 0) {
+        setToggled(false);
+      }
+    } else {
+      selectedRedhatRepos.add(uuid);
+    }
+    setSelectedRedhatRepos(new Set(selectedRedhatRepos));
+  };
+
+  const [searchQuery, setSearchQuery] = useState('');
+  const debouncedSearch = useDebounce(searchQuery);
+
+  const [page, setPage] = useState(1);
+  const [perPage, setPerPage] = useState(20);
+  const [activeSortIndex, setActiveSortIndex] = useState<number>(0);
+  const [activeSortDirection, setActiveSortDirection] = useState<'asc' | 'desc'>('asc');
+
+  const onSetPage = (_, newPage: number) => setPage(newPage);
+  const onPerPageSelect = (_, newPerPage: number, newPage: number) => {
+    setPerPage(newPerPage);
+    setPage(newPage);
+  };
+
+  const columnHeaders = ['Name', /* 'Label',*/ 'Advisories', 'Packages'];
+
+  const columnSortAttributes = ['name', '', '', 'package_count'];
```

Comment:
should both the name and packages column be sortable for the RH repositories step? looks like it's only sortable by name